pkg/sqs/sqs.go

```diff
@@ -99,24 +110,20 @@ func (s *AWSSqsSource) Read(_ context.Context, readRequest sourcesdk.ReadRequest
 		case <-ctx.Done():
 			return
 		default:
-			s.lock.Lock()
 			messageCh <- sourcesdk.NewMessage(
```

Comment:
is the mutex lock not necessary here ?

Reply:
Yes. It was introduced to protect the toAckSet.